helm/hpcc/templates/_helpers.tpl

```diff
@@ -139,6 +139,33 @@ Returns the largest number of workers from all the thors
  {{- $maxNumWorkers -}}
 {{- end -}}
 
+{{/*
+Returns true if the given certificate issuer is enabled, otherwist false
+*/}}
+{{- define "hpcc.isIssuerEnabled" -}}
+{{- $certificates := (.root.Values.certificates | default dict) -}}
+{{- $issuers := ($certificates.issuers | default dict) -}}
+{{- $issuer := get $issuers .issuer -}}
+{{- if ($issuer) -}}
+  {{ (hasKey $issuer "enabled" | ternary $issuer.enabled true) }}
```

Comment:
I think it's okay, because of the space clipping at end of it, otherwise would return "  true" and fail to match.
Maybe should use {{- on this line to be clearer?